### PR TITLE
Upgrade to Angular 8

### PR DESCRIPTION
**Pull request by dmayerdesign**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,44 +16,43 @@
...
-    "@angular/forms": "^7.2.0",
-    "@angular/http": "^7.2.0",
-    "@angular/platform-browser": "^7.2.0",
-    "@angular/platform-browser-dynamic": "^7.2.0",
-    "@angular/router": "^7.2.0",
-    "core-js": "2.5.4",
-    "lodash": "4.17.11",
-    "rxjs": "^6.5.1",
-    "zone.js": "^0.8.29"
+    "@angular/animations": "^8.2.2",
```
Comment (thomasbottonari) [inline package.json]: what is the best practice for public modules? seems like you don't want to be as specific as 8.2.2 but rather anything 8 or above, right?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "rxjs-uploader-demo",
   "version": "1.4.1",
```
Comment (thomasbottonari) [inline package.json]: you intentionally didn't update this, right?

File: package.json (hunk 1 above)
Comment (dmayerdesign) [reply package.json]: So the way it's organized is really confusing—think of this `package.json` as the "development" one, and the one in `projects/rxjs-uploader` as the real one. Basically, this one only matters for building stuff, and all the angular deps are just for the demo app. I think this sort of versioning is fine here.

File: package.json (hunk 2 above)
Comment (dmayerdesign) [reply package.json]: Correcto, Matt did it in a previous PR—we'll bundle all these changes into the same release.

File: package.json (hunk 1 above)
Comment (thomasbottonari) [reply package.json]: right, my mistake. i didn't look at which package.json it was.

**Pull request merged**